docs/analyse/koble-pseudonymiserte.md

```diff
@@ -0,0 +1,17 @@
+# Koble sammen pseudonymiserte views
```

Comment:
Ganske sikker på at du får to headers nå, hvis det er ønskelig så er to `#` riktig, da dette kun er en hoved header per side.